cdap-examples/UserProfiles/bin/add-users.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+#
+# Copyright © 2014 Cask Data, Inc.
```

Comment:
done, everywhere, Looks like the IDEA copyright plugin does not work for shell scripts...